sudoku/sudoku.rb

```diff
@@ -0,0 +1,88 @@
+class Sudoku
+  def initialize(board)
+    @board = board
```

Comment:
maybe it could be named as `@rows` since we have a method that returns just it ?